example/styles/html-example.css

```
@@ -0,0 +1,5 @@
+@import url('./partials/reset.css');
```

Stylelint (17.4.0)

[error] Expected "url('./partials/reset.css')" to be "'./partials/reset.css'" (import-notation)

(import-notation)